src/scripts.js

```diff
@@ -150,13 +151,88 @@ let stepsTrendingButton = document.querySelector('.steps-trending-button');
 let stepsUserStepsToday = document.querySelector('#steps-user-steps-today');
 let trendingStepsPhraseContainer = document.querySelector('.trending-steps-phrase-container');
 let trendingStairsPhraseContainer = document.querySelector('.trending-stairs-phrase-container');
-let userInfoDropdown = document.querySelector('#user-info-dropdown');
+let userInfoDropdown = document.querySelector('.user-info-dropdown');
+let sleepDate = document.querySelector('.date-input');
+let hoursSleptInput = document.querySelector('.hours-slept-input');
+let sleepQualityInput = document.querySelector('.sleep-quality-input');
+let submitButton = document.querySelector('.submit-button');
+let hydrationDateInput = document.querySelector('.hydration-date-input');
+let ouncesDrankInput = document.querySelector('.ounces-drank-input');
+let hydrationSubmitButton = document.querySelector(".hydration-submit-button");
+let activityDateInput = document.querySelector(".activity-date-input");
+let numberOfStepsInput = document.querySelector(".number-steps-input");
+let minutesActiveInput = document.querySelector(".minutes-active-input");
+let flightsOfStairsInput = document.querySelector(".stairs-input");
+let activitySubmitButton = document.querySelector(".activity-submit-button");
+let showSleepFormButton = document.querySelector(".show-sleep-form");
+let showActivityFormButton = document.querySelector(".show-activity-form");
+let showHydrationFormButton = document.querySelector(".show-hydration-form");
+let sleepInputForm = document.querySelector(".sleep-input-form");
+let activityInputForm = document.querySelector(".activity-input-form");
+let hydrationInputForm = document.querySelector(".hydration-input-form");
+let successfulSubmit = document.querySelector(".successful-submit");
+let failedSubmit = document.querySelector(".failed-submit");
+let clearButton = document.querySelector(".clear-button");
+let errorMessage = document.querySelector(".error-message");
 
 mainPage.addEventListener('click', showInfo);
 profileButton.addEventListener('click', showDropdown);
 stairsTrendingButton.addEventListener('click', updateTrendingStairsDays());
 stepsTrendingButton.addEventListener('click', updateTrendingStepDays());
 
+submitButton.addEventListener('click', postSleepHelper );
+hydrationSubmitButton.addEventListener('click', postHydrationHelper);
+activitySubmitButton.addEventListener('click', postActivityHelper);
+showSleepFormButton.addEventListener('click', showSleepForm);
+showActivityFormButton.addEventListener('click', showActivityForm);
+showHydrationFormButton.addEventListener('click', showHydrationForm);
```

Comment:
Could possibly use event delegation for this??

Reply:
Defiantly, I was thinking we do this in the script refactor.